src/components/PostContent.astro

```diff
@@ -19,6 +19,37 @@
     font-size: 1.25em;
   }
 
+  div > :global(table) {
```

Comment:
Is there a specific reason for why we have the `div >` here?

Reply:
It targets the `div` above and scopes the styles to that

Reply:
Ok, I see. Thakns

Reply:
To explain a bit more: Astro will find the `div` in the HTML at the top of this component and generate a CSS class name for it. The styles in the `style` tag will be applied to that class name. The `:global()` makes sure that Astro isn't also generating classes for each targeted component after that in the selector, eg. it generates `.divClass > table`.

If we would just use `div > table` it would generate `.divClass > .tableClass`. Because we are injecting the content here rather than rendering it statically, Astro isn't able to attach that `.tableClass` to table elements on the page and so the styles would never apply, hence the `:global(…)`

Reply:
Thank you very much! I am not too familiar with CSS Modules, so your explanation helped a lot